docs/user-guide/src/bmo/live-iso.md

```diff
@@ -0,0 +1,32 @@
+# Live-ISO
+
+The live-iso API in Metal3 allows to boot BaremetalHost with live-iso image instead of using IPA deploy ramdisk.
+
+Why we need it?
+
+In some circumstances, i.e to reduce boot time for ephemeral workloads, it may be possible to boot an iso
+and not deploy any image to disk (saving the time to write the image and reboot). Since this alternative
+is distributed as a live-iso, leveraging the existing toolchains like [fedora-coreos installer](https://docs.fedoraproject.org/en-US/fedora-coreos/bare-metal/) might be desirable.
+
+How to use it?
+
+Here is an example with a BaremetalHost CRD, where iso referenced by the url and `live-iso` set in DiskFormat will be
+live-booted without deploying an image to disk. Checksum options are not required in this case, and will be ignored if specified:
+
+```yaml
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: live-iso-booted-node
+spec:
+  image:
+    url: http://1.2.3.4/image.iso
+    format: live-iso
+  online: true
+```
+
+**Note**  `rootDeviceHints` and `userData` will not be used
+since the image won't be written to disk, and Ironic doesn't
+support passing user-data in addition to the iso attachment at present.
```

Comment:
It's not 100% correct, Ironic supports attaching it as a virtual USB stick. The problem is, a lot of hardware does not allow 2 virtual media devices at the same time. I'd probably skip the reasoning. But do mention that network data is also not used.

Reply:
I have removed the reasoning part for ironic, however did leave that `rootDeviceHints`, `networkData` and `userData` won't be used due to it image is not written to the disk. Hope that makes sense 🙂